Brewfile

```diff
@@ -48,6 +49,8 @@ tap "heroku/brew"
 brew "tfenv"
 brew "tflint"
 brew "heroku/brew/heroku"
+brew "awscli"
+brew "kubernetes-cli"
```

Comment:
You run these yourself? Do we use kubernetes anywhere?

Reply:
I can get behind `awscli` but probably not Kubernetes unless we're actually using it at all.

Reply:
Err sorry, we don't use Kubernetes, I definitely need another coffee.